### PR TITLE
Correspondence of issue # 45

### LINKED ISSUE #45: Cannot delete external cell from Community Screen
cannot_delete_cell

### PR DESCRIPTION
**Pull request by csKyou**
- I stopped obtaining relation name and box name from id and passed it as an argument.
 - Some arguments of sg.appendRelationLinkExtCellAfter, sg.appendRelationLinkExtCell function wrapped
close #45

### CODE REVIEW

File: appcell-resources/js/socialgraph.js [hunk 1]
```diff
@@ -197,21 +195,23 @@ sg.dispExtCellLinkRelation = function(json, extUrl) {
...
-      var dispName = sg.getName(extUrl);
-      var description = "";
-      var imageSrc = cm.notImage;
-      var extRelID = "dvExtCellRelList";
+      var profObj = {
+          DisplayName: sg.getName(extUrl),
+          Description: "",
+          Image: cm.notImage
+      }
+      var extRelObj = {ID: "dvExtCellRelList"};
```
Comment (dixonsiu) [inline appcell-resources/js/socialgraph.js]: When you assign "dvExtCellRelList" here, would you be able to get it back in line 227 & 230

File: appcell-resources/js/socialgraph.js [hunk 2]
```diff
@@ -222,82 +222,91 @@ sg.dispRelationLinkExtCell = function(json, relName, relBoxName) {
...
-            boxName = splitID[2];
+        if (extRelObj.BoxName) {
+            boxName = extRelObj.BoxName;
         }
-        html = '<a class="del-button list-group-item" style="top:25%" href="#" onClick="sg.dispDelExtCellRelationModal(\'' + url + '\',\'' + relName + '\',\'' + boxName + '\');return(false)" data-i18n="Del"></a>';
+        var anotherTag = $('<a>', {
+            class: 'del-button list-group-item', 
+            style: 'top:25%;', 
+            onClick: 'sg.dispDelExtCellRelationModal(this)',
+            'data-i18n': 'Del'
```
Comment (dixonsiu) [inline appcell-resources/js/socialgraph.js]: I think data-i18n is not allowed, the initialization only allow HTML standard attributes.

Comment (csKyou) [reply appcell-resources/js/socialgraph.js]: For non-standard attributes, you can specify it with a single quote.

File: appcell-resources/js/socialgraph.js (hunk 1 above)
Comment (csKyou) [reply appcell-resources/js/socialgraph.js]: This element is the default section, therore, the ID is always "dvExtCellRelList".

**Pull request merged**